packages/server/config.ts

```diff
@@ -1,7 +1,20 @@
-require("dotenv").config();
-require("dotenv").config({ path: "./.env.local" });
+require("dotenv").config({ path: "./.env" });
+
+interface Config {
+  database_client: string;
+  database_url: string;
+  token: {
+    token_secret: string;
+    expiresIn: number;
+  };
+}
 
 export const config = {
-  DATABASE_CLIENT: "postgres",
-  DATABASE_URL: process.env.DATABASE_URL,
+  database_client: "postgres",
+  database_url: process.env.DATABASE_URL,
+  token: {
+    expiresIn: parseInt(process.env.TOKEN_EXPIRY, 10),
+    length: parseInt(process.env.TOKEN_LENGTH, 10),
+    alphabet: process.env.TOKEN_ALPHABET,
```

Comment:
Do you use it ?
```suggestion
```

Reply:
Bien vu! C'était avec un autre package pour créer des token